__tests__/server/utils/createCircuitBreaker.spec.js

```diff
@@ -64,6 +65,19 @@ describe('Circuit breaker', () => {
     expect(value).toBe(true);
   });
 
+  it('should not open the circuit when in development environment', async () => {
+    process.env.NODE_ENV = 'development';
+    expect.assertions(2);
+    setEventLoopDelayThreshold(-1);
```

Comment:
This is to flip the switch on the circuit breaker every time right?

Reply:
yes it is, so that we can ensure it is would be running